test/shaderdb/fuzzer_found/GraphicsFuzz_FragmentShader_CallIfWhileSwitch.spvas

```diff
@@ -0,0 +1,104 @@
+; BEGIN_SHADERTEST
```

Comment:
Thanks for adding those assembly tests. Please change the directory and test name
```
shaderdb/fuzzer_found/GraphicsFuzz_FragmentShader_CallIfWhileSwitch.spvas
->
shaderdb/fuzzer/GraphicsFuzz_TestCallIfWhileSwitch.spvas
```
(1) Create a "fuzzer" fold is OK. Although the issues are found, we actually don't need to mention this as folder name ;)
(2) The case name style is often this: OpXXX_TestYYY, ExtXXX_TestYYY. So for all cases, don't mention the shader stages unless we do need this info because of stag-dependent implementation. Please add "Test" as well to following existing case name style.

Reply:
Thanks for the suggestion. I renamed all the files.